pylintrc

```diff
@@ -240,7 +240,8 @@ generated-members=REQUEST,acl_users,aq_parent
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=100
+max-line-length=125
```

Comment:
125 is the standard of what is 'diffable' on github, default 100 is too restrictive